rfc/rfc006-distributed-registry.md

```diff
@@ -342,6 +342,37 @@ The care organisation refers to it:
 }
 ```
 
+### 4.1. Contact information
+
+A node's DID document SHOULD define a service of type `node-contact-info` that contains information that can be used
```

Comment:
The other specs do not yet speak of a DID Document belonging to a node. So in this case it could be: any DID Document may specify a `node-contact-info`.